BrightID/src/actions/fetchUserInfo.ts

```diff
@@ -23,67 +17,29 @@ const fetchUserInfo = (api: NodeApi) => (
     InteractionManager.runAfterInteractions(async () => {
       const {
         user: { id },
-        groups: { invites: oldInvites },
       } = getState();
       // const api = selectNodeApi(getState());
 
-      const opTotal = selectOperationsTotal(getState());
-      console.log('opTotal', opTotal);
-
       console.log('refreshing user info', id);
       if (!id) {
         throw new Error('id missing');
       }
       try {
-        const {
-          groups,
-          score,
-          connections = [],
-          isSponsored,
-          invites,
-        } = await api.getUserInfo(id);
-
-        const verifications = await api.getUserVerifications(id);
-
+        const verifications = await api.getVerifications(id);
+        dispatch(setVerifications(verifications));
+        const memberships = await api.getMemberships(id);
+        dispatch(updateMemberships(memberships));
+        const connections = await api.getConnections(id, 'outbound');
         let incomingConns = await api.getConnections(id, 'inbound');
-
         incomingConns = _.keyBy(incomingConns, 'id');
-
         for (const conn of connections) {
           conn.incomingLevel = incomingConns[conn.id]?.level;
         }
+        dispatch(updateConnections(connections));
 
-        if (opTotal === 0) {
```

Comment:
thanks
I thought we check state and remove connection only if it is currently in confirmed (not pending) state but it's not in the server response.
I thought this `if` block is because we did not have `state` for groups to monitor them like connections and now that `state` is added to groups, there is no problem anymore and we can remove this `if`.

Reply:
We are still setting newly created groups to "dismissed" when they are not part of the server response (see `updateMemberships()` in groupsSlice.ts). I re-added the protection just for group operations, but for a real fix we should update groupsSlice to properly handle newly created groups that only exist on the client.